### ISSUE TITLE
Spacemacs not loaded on Windows, unless started from home dir in Bash shell

### ISSUE THREAD

**Issue by mrotaru** [syl20bnr/spacemacs#14073]
I installed Emacs with [scoop](https://scoop.sh/), but when I start Emacs from the start menu, it loads the "vanilla" version, without Spacemacs. I have my configuration in `~/.spacemacs`, and `~/.emacs.d` is a clone of the development branch. 

If I start it by executing `emacs` from `git-bash`, and I'm in the home directory, it will launch with the Spacemacs config loaded.
If I start it by executing `emacs` from `git-bash`, and I'm **not** in the home directory, it will launch with the Spacemacs config loaded but will show a warning; in the example below, I was in ~/Documents:
```
Warning (initialization): An error occurred while loading ‘c:/Users/Mihai/.emacs.d/init.el’:

File is missing: Setting current directory, No such file or directory, ~/Documents/

To ensure normal operation, you should investigate and remove the
cause of the error in your initialization file.  Start Emacs with
the ‘--debug-init’ option to view a complete error backtrace.
```

How can I configure Emacs, such that Spacemacs is loaded when I start it using the link in the start menu, and from any location in the `git-bash` shell ?

**Comment by duianto** [comment]
>when I start Emacs from the start menu, it loads the "vanilla" version, without Spacemacs. I have my configuration in ~/.spacemacs, and ~/.emacs.d is a clone of the development branch.

By default Emacs looks for the `.emacs.d` directory in:
`C:\Users\username\AppData\Roaming`
source:
https://www.gnu.org/software/emacs/manual/html_node/emacs/Windows-HOME.html

If you want to put them in your home directory: 
`c:/users/Mihai`

Then you'll have to setup a `HOME` environment variable with the value of your home directory.

I used to have a `HOME` variable setup, but it was removed to test another issue:
Can't quit Emacs #14051
(I didn't encounter that issue either with or without the `HOME` variable)

I've installed Emacs with the official installer:
https://www.gnu.org/software/emacs/

Which installed it in: `C:\Program Files\Emacs\`

My start menu shortcut points to:
`"C:\Program Files\Emacs\x86_64\bin\runemacs.exe"`

Here's an explanation about the difference between `emacs` and `runemacs`:
https://lists.gnu.org/archive/html/help-gnu-emacs/2004-03/msg00168.html
>emacs.exe works in both console and GUI mode. To do this, it is
marked as a console program as far as Windows is concerned, so
Windows will open a command-prompt window when it is launched from
Explorer.
>
>runemacs.exe is a GUI program, so no ugly console, and its purpose is
to launch emacs.exe with the command-prompt window hidden.
